.github/workflows/backend_deploy.yaml

```
@@ -77,6 +77,7 @@ jobs:
           key: ${{ secrets.SSH_PRIVATE_KEY }}
           passphrase: ${{ secrets.PASSPHRASE }}
           script: |
+            cd ../../usr/local/src/
```

Если ты в корень лезешь, то лучше обращаться так
```suggestion
            cd /usr/local/src/
```

---

Хотя, в чём смысл? ты сначала идёшь в какую-то папку, а потом сразу в другую, выглядит так, что этот код лишний

---

Верно и для другого файла